sankaku/constants.py

```diff
@@ -1,8 +1,9 @@
 """Necessary constants such as hardcoded headers, API urls and endpoints,
 default values of parameters etc.
 """
+from typing import Dict
```

Comment:
There should be a blank line after module docstring to keep code style consistency.